src/main/java/com/puppycrawl/tools/checkstyle/checks/ClassResolver.java

```diff
@@ -184,6 +184,7 @@ public ClassResolver(ClassLoader loader, String pkg, Set<String> imports) {
     }
 
     /**
+     * Returns true if specified class is loadable with safeLoad() or false.
```

Comment:
Again, describe what the method does. What you have written could just go directly in the `@return` description.
Change to something like `Checks if the given class name can be loaded.`.